lib/susepubliccloudinfoclient/infoserverrequests.py

```diff
@@ -319,14 +343,16 @@ def get_provider_data(
         type,
         apply_filters=command_arg_filter
     )
-    return __process(url, info_type, command_arg_filter, result_format)
+    return __process(url, info_type, command_arg_filter,
+                     result_format, transfer_compression)
 
 
 def get_image_states_data(
         framework,
         type,
         result_format='plain',
         region='all',
+        transfer_compression='bz2',
```

Comment:
This breaks the API for every client that used positional arguments, i.e. possible effects a lot of our automation tooling. and we do not know about customer tooling, the package is public. The new argument must be added to the end of the argument list.